scripts/upgrade

```diff
@@ -166,6 +166,9 @@ sub_path_only="$(if [[ "$path_url" == "/" ]]; then echo '# ' ; else echo ''; fi)
 ynh_add_config --template ../conf/ihatemoney.cfg --destination "$final_path/ihatemoney.cfg"
 chmod 600 "$final_path/ihatemoney.cfg"
 
+# Upstream issue : https://github.com/spiral-project/ihatemoney/issues/973
+src_dir=$("$final_path/venv/bin/python3" -c "import ihatemoney ; print(ihatemoney.__file__)")
+patch -p1 -d "$(dirname "$src_dir")" < "../conf/ihatemoney.patch"
```

Comment:
```suggestion
patch -p1 -d "$(dirname "$src_dir")" < "../conf/ihatemoney_showcase.patch"
```

Reply:
Also, if you put your patch in` sources -> patches` folder, the patch should be applied automatically after source install.

Reply:
I won't put it in the patches folder, because I don't use ynh_setup_source. ihatemoney is installed by pip in the venv :)

Reply:
Good catch for the filename though, I'm fixing it right now.